mac-build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This should not be hard coded to bash.

Reply:
My changes were based off of the iOS build script, which also uses a hard-coded Bash. I'll change it to just plain sh.

Reply:
I only noticed the script that was being replaced was `/bin/sh`. I believe osx is now using zsh by default?

Reply:
As of 10.15 (Catalina), yes.